core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -134,7 +134,11 @@ public Application(T config, String path, String listenerName) {
     logWriter.setLoggerName(config.getString(RestConfig.REQUEST_LOGGER_NAME_CONFIG));
 
     // %{ms}T logs request time in milliseconds
-    requestLog = new CustomRequestLog(logWriter, CustomRequestLog.EXTENDED_NCSA_FORMAT + " %{ms}T");
+    requestLog = new CustomRequestLog(logWriter, requestLogFormat());
+  }
+
+  protected String requestLogFormat() {
+    return CustomRequestLog.EXTENDED_NCSA_FORMAT + " %{ms}T";
```

Comment:
LGTM....

But be careful how you override this. The request log format is part of the public "API" of the application, as in, there are people out there that depend on it (on on-prem).

Reply:
Thanks for the reminder -- we plan to only override this in our cloud deployment